README.md

```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
+| <img src="https://avatars1.githubusercontent.com/u/5222525?s=460&v=4" width="100" /> | Antoine Eddi ([@aeddi](https://github.com/aeddi)) | Implementing libp2p over Bluetooth Low Energy | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-bluetoothle.pdf) | [Video](https://youtu.be/aaSFHxpwm9A) |
+| <img src="https://avatars1.githubusercontent.com/u/21236?s=460&v=4" width="100" /> | Irakli Gozalishvili ([@gozala](https://github.com/gozala))  | Inter Planetary Data Feed (IPDF) | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-ipdf.pdf) | [Video](https://youtu.be/KBwR0I7i4Wg) |
+| <img src="https://avatars1.githubusercontent.com/u/1398860?s=460&v=4" width="100" /> | Gonçalo Pestana ([@gpestana](https://github.com/gpestana)) | Privacy in P2P networks: DHTs and IPFS | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-privacyp2p.pdf) | [Video](https://youtu.be/lGk6OErtDFY) |
+| <img src="https://avatars2.githubusercontent.com/u/81581?s=460&v=4" width="100" /> | Victor Grishchenko ([@gritzko](https://github.com/gritzko))  | An overview of RON | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-ron.pdf) | [Video](https://youtu.be/KJTGv1h-nUM) |
+| <img src="https://avatars3.githubusercontent.com/u/248257?s=460&v=4" width="100" /> | Rüdiger Klaehn ([@rklaehn](https://github.com/rklaehn))  | Actyx's experiences on building with IPFS | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-actyx.pdf) | [Video](https://youtu.be/Qzu0xtCT-R0) |
+| <img src="https://avatars3.githubusercontent.com/u/11004800?s=460&v=4" width="100" /> | Jonathan Budd ([@cesarosum](https://github.com/cesarosum)) | Peergos: E2E encrypted storage and social network on IPFS | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-peergos.pdf) | [Video](https://youtu.be/MBHsJlx7knA) |
+| <img src="https://avatars3.githubusercontent.com/u/1017236?s=460&v=4" width="100" /> | André Cruz ([@satazor](https://github.com/satazor)) | Introducing Nomios | [Website](https://nomios.io/) | [Video](https://youtu.be/EEIq6Eu9ug4) |
+| <img src="https://avatars0.githubusercontent.com/u/17089485?s=460&v=4" width="100" /> | Alexandre Trottier ([@postables](https://github.com/postables)) | Why open source matters and how individuals can further the movement | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-opensource.pdf) | [Video](https://youtu.be/usa796ebEnI) |
+| <img src="https://avatars3.githubusercontent.com/u/370259?s=460&v=4" width="100" /> | Andrew Hill ([@andrewxhill](https://github.com/andrewxhill)) | Textile | SLIDES | [Video](https://youtu.be/uI9eEZQZpfw)
```

Comment:
@andrewxhill do you happen to have an url to the slides of your Textile Lighting Talk?

Reply:
just seeing this. yep, they are already committed, https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/Andrew_Hill_IPFS_Personal_data_ownership.pdf

Reply:
oh, looks like you got them, cheers!

Reply:
I meant the second talk you did, the one in the last day -> https://www.youtube.com/watch?v=uI9eEZQZpfw